boards/shared/Src/App/App_SharedErrorTable.c

```diff
@@ -54,6 +55,8 @@ struct ErrorTable *App_SharedErrorTable_Create(void)
     INIT_ERROR(FSM_NON_CRITICAL_RIGHT_WHEEL_SPEED_OUT_OF_RANGE, FSM, false);
     INIT_ERROR(FSM_NON_CRITICAL_PRIMARY_FLOW_RATE_OUT_OF_RANGE, FSM, false);
     INIT_ERROR(FSM_NON_CRITICAL_SECONDARY_FLOW_RATE_OUT_OF_RANGE, FSM, false);
+    INIT_ERROR(FSM_NON_CRITICAL_STEERING_ANGLE_OUT_OF_RANGE, FSM, false);
```

Comment:
With the changes mentioned [here](https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/642/files#r436227848), I was able to identify errors I forgot to add (1 from DIM and 2 from FSM). This is the kind of human-error I wanted to avoid with the help of the codegen.